deb/after_install.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+#
+# Script which is run after install to ensure binary compatibility between
+# this and the previous version of libdrizzle-redux
+
+if [ -h /usr/lib/libdrizzle-redux.so.9 ]; then
+    echo "Removing outdated symlink"
+    sudo rm /usr/lib/libdrizzle-redux.so.10 /usr/lib/libdrizzle-redux.so.9 2>/dev/null
+    if [ -e "/usr/lib/libdrizzle-redux.so.9" ]; then
+        echo "Couldn't remove symlink '/usr/lib/libdrizzle-redux.so.9'"
+              "Please remove it manually"
```

Comment:
Some might want to install the package as part of automated install (using `apt-get install -y`), whatever choice you go with just make sure that your script plays nicely with `-y`.